src/ioctls/device.rs

```diff
@@ -80,6 +80,21 @@ impl DeviceFd {
         }
         Ok(())
     }
+
+    /// Gets a specified piece of device configuration and/or state.
+    ///
+    /// See the documentation for `KVM_GET_DEVICE_ATTR`.
+    /// # Arguments
```

Comment:
Also add a `# Returns` section so we can add details such as this ioctl returns its result in the input parameter (and what fields should be used as return).

Reply:
Done.